plugin/storage/badger/config.go

```diff
@@ -42,6 +42,9 @@ type Config struct {
 	// ReadOnly opens the data store in read-only mode. Multiple instances can open the same
 	// store in read-only mode. Values still in the write-ahead-log must be replayed before opening.
 	ReadOnly bool `mapstructure:"read_only"`
+	// DualLookUp enables the look-up in old indexes of jaeger which are not deprecated.
+	// By-default it is enabled and it is suggested not to disable it
+	DualLookUp bool `mapstructure:"dual_look_up"`
```

Comment:
let's not expose this in the config. We should use a feature gate instead - see https://github.com/jaegertracing/jaeger/pull/6568